specification/automation/resource-manager/Microsoft.Automation/stable/2018-06-30/python2package.json

```diff
@@ -0,0 +1,733 @@
+        },
+        "name": {
+          "type": "string",
+          "description": "Gets or sets name of the resource."
+        },
+        "location": {
+          "type": "string",
+          "description": "Gets or sets the location of the resource."
+        },
+        "tags": {
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          },
+          "description": "Gets or sets the tags attached to the resource."
+        }
+      },
+      "required": [
+        "properties"
+      ],
+      "description": "The parameters supplied to the create or update module operation."
+    },
+    "ModuleProperties": {
+      "properties": {
+        "isGlobal": {
+          "type": "boolean",
+          "description": "Gets or sets the isGlobal flag of the module."
+        },
+        "version": {
+          "type": "string",
+          "description": "Gets or sets the version of the module."
+        },
+        "sizeInBytes": {
+          "type": "integer",
+          "format": "int64",
+          "description": "Gets or sets the size in bytes of the module."
+        },
+        "activityCount": {
+          "type": "integer",
+          "format": "int32",
+          "description": "Gets or sets the activity count of the module."
+        },
+        "provisioningState": {
+          "type": "string",
+          "description": "Gets or sets the provisioning state of the module.",
+          "enum": [
+            "Created",
+            "Creating",
+            "StartingImportModuleRunbook",
+            "RunningImportModuleRunbook",
+            "ContentRetrieved",
+            "ContentDownloaded",
+            "ContentValidated",
+            "ConnectionTypeImported",
+            "ContentStored",
+            "ModuleDataStored",
+            "ActivitiesStored",
+            "ModuleImportRunbookComplete",
+            "Succeeded",
+            "Failed",
+            "Cancelled",
+            "Updating"
+          ],
+          "x-ms-enum": {
+            "name": "ModuleProvisioningState",
+            "modelAsString": false
+          }
+        },
+        "contentLink": {
+          "$ref": "#/definitions/ContentLink",
+          "description": "Gets or sets the contentLink of the module."
+        },
+        "error": {
+          "$ref": "#/definitions/ModuleErrorInfo",
+          "description": "Gets or sets the error info of the module."
+        },
+        "creationTime": {
+          "type": "string",
+          "format": "date-time",
+          "description": "Gets or sets the creation time.",
+          "x-nullable": false
+        },
+        "lastModifiedTime": {
+          "type": "string",
+          "format": "date-time",
+          "description": "Gets or sets the last modified time.",
+          "x-nullable": false
+        },
+        "description": {
+          "type": "string",
+          "description": "Gets or sets the description."
+        },
+        "isComposite": {
```

Comment:
same comment for bool

Reply:
isComposite is never expected to change. The specs for Python 2 packages are practically the same a Powershell modules for consistency, except for paths.